src/ipc-handlers.ts

```diff
@@ -388,7 +388,17 @@ export async function startServer(
 	await keepSqliteIntegrationUpdated( server.details.path );
 
 	const parentWindow = BrowserWindow.fromWebContents( event.sender );
-	await server.start();
+	try {
+		await server.start();
+	} catch ( error ) {
+		if (
+			error instanceof Error &&
+			error.message.includes( '"unreachable" WASM instruction executed' )
+		) {
+			throw new Error( 'Please try disabling plugins and themes that might be causing the issue.' );
```

Comment:
We could translate the error message here with `__`, but we probably prefer to log, and capture in Sentry all the messages in English.

Reply:
Should we move the Sentry call here, too, so we would log it before we rethrow the exception?

Reply:
@wojtekn , thanks for the suggestion. It makes sense. I've moved it here:

https://github.com/Automattic/studio/pull/753/commits/9c301a24a5a92e2699c37bc97018009006d771c2